flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/to_source/comment1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Sample diagram node",
+  "diagram": {
+    "id": "32802",
+    "metadata": {
+      "label": "Comment",
+      "description": "Comment of a node"
+    },
+    "codedata": {
+      "node": "COMMENT",
+      "lineRange": {
+        "fileName": "comment.bal",
+        "startLine": {
+          "line": 1,
+          "offset": 0
+        },
+        "endLine": {
+          "line": 3,
+          "offset": 4
+        }
+      }
+    },
+    "returning": false,
+    "properties": {
+      "comment": {
+        "metadata": {
+          "label": "Comment",
+          "description": "Comment to describe the flow"
+        },
+        "value": "    // Create variable name i\n    // This is an additional comment\n    ",
```

Comment:
Consequently, the input should not contain `//`. The source generator should add this explicitly.

Reply:
Fixed